### PR TITLE
Re-adding the Salvage Shuttle

### PR DESCRIPTION
**Pull request by Mangohydra**
## **ITS BACK!**
The reclaimer has returned, with a few dents and holes you'll have to patch up.

To be more specific, the reclaimer and salvage shuttle console board/computer have been re-added.
You can also find a spare board in the QM's locker, just remember to ask first.

## Why / Balance
I was asked very nicely.

## Media
The (NEW) reclaimer _(posters shown are randomized)_
image
Salvage shuttle console
image
Board found in the QM's locker
image

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
- The reclaimer has returned.
- Salvage shuttle boards and computers exist again.
- Qm lockers come with 1 spare salvage shuttle console board.

### CODE REVIEW

**Review by dffdff2423 — CHANGES_REQUESTED** [review]
Please fix the merge conflicts before we review it.

File: Resources/migration.yml [hunk 1]
```diff
@@ -386,9 +386,7 @@ ClothingBeltSuspenders: ClothingBeltSuspendersRed
 ClothingNeckShockCollar: ClothingBackpackElectropack
 
 # 2024-08-22
-ComputerShuttleSalvage: null
-SalvageShuttleConsoleCircuitboard: null
-SalvageShuttleCircuitboardStealObjective: null
```
Comment (dffdff2423) [inline Resources/migration.yml:391]: Just comment instead of deleting them. However, the maps have likely been re-saved upstream since this change so you will need to make map patches for upstream maps and ensure they are mapped on any CD maps.

**Review by dffdff2423 — CHANGES_REQUESTED** [review]

**Comment by dffdff2423** [comment]
Given that the salvage shuttle is now CD content it should be namespaced. Keep the upstream map directory as is.

File: Resources/migration.yml (hunk 1 above)
Comment (dffdff2423) [reply Resources/migration.yml:391]: I am actually not sure that map patches are needed, it might be better to not include the console on the station to drive home the fact it is really run down.

Comment (Mangohydra) [reply Resources/migration.yml:391]: I've made them commented, should be good now.

**Comment by Mangohydra** [comment]
> Given that the salvage shuttle is now CD content it should be namespaced. Keep the upstream map directory as is.

Its now in the CD namespace with a shuttles folder.
Also the conflict is resolved github just isn't happy about it for some reason??

**Comment by Mangohydra** [comment]
what the fuck is this
image

**Comment by dffdff2423** [comment]
> what the fuck is this image

The merge conflict

**Comment by Mangohydra** [comment]
> The merge conflict. I don't see it on github so you probably are in the middle of your rebase/merge. Running `git status` should give you a bit more info.

i got nothing of note from it, i'm just going to remove the console board from the locker and then put it back, maybe that will fix it?

**Comment by dffdff2423** [comment]
See upstream's guide on resolving merge conflicts: https://docs.spacestation14.com/en/general-development/setup/git-for-the-ss14-developer.html?highlight=merge%20conflict#resolving-merge-conflicts

TLDR: 
You probably just need to do the following (assuming you are using the git cli aka gitbash, I do not know how it works with gui git clients):
- abort whatever operation you are currently doing (see the result of `git status`)
- Run `git fetch <remote>` where `<remote>` is a remote that points to CD's repo (not your fork)
- Run `git rebase <remote>/master` on your branch
- Follow the listed instructions to resolve any merge conflicts
- `git push origin <your-branch> --force`

If you need to be walked through it, feel free to ask on discord.

**Comment by Mangohydra** [comment]
got lost at the remote part, too eepy

**Comment by Mangohydra** [comment]
I've done some localhost tests, something has gone apeshit and now everything is fucked, rounds will no longer start and instead instantly restart.
the 10 hells are upon me

**Comment by Mangohydra** [comment]
the branch might be fucked

**Comment by Mangohydra** [comment]
I'll re-open this once I have time to fix this, or if I feel like doing it at all, which I'm still uncertain on at this point.

**Comment by Mangohydra** [comment]
Also 100% am allowing anyone to take over this branch and find the demon thats haunting it.
***IF*** *i work on this again it'll be a fresh branch to prevent issues like this from happening again.*

**Comment by dffdff2423** [comment]
Feel free to ask for help on discord if you are stuck.
